src/foam/nanos/notification/NotificationView.js

```diff
@@ -8,43 +8,124 @@ foam.CLASS({
   package: 'foam.nanos.notification',
   name: 'NotificationView',
   extends: 'foam.u2.View',
+  documentation: 'Notification controller',
 
-  axioms: [
-    foam.pattern.Faceted.create()
+  implements: [
+    'foam.mlang.Expressions'
   ],
 
+  imports: [
+    'currentMenu',
+    'group',
+    'notificationDAO',
+    'stack',
+    'user'
+  ],
+
+  requires: [
+    'foam.nanos.notification.Notification',
+    'foam.nanos.notification.NotificationRowView',
+    'foam.u2.stack.Stack'
+  ],
+
+  exports: [
+    'as data',
```

Comment:
Should extend Controller and then remove this line if this is a Controller not a View